tests/ci/sync-to-aws.sh

```diff
@@ -16,6 +16,13 @@ echo "[+] Configure AWS settings"
 $AWS_CLI configure set default.region "$AWS_REGION"
 $AWS_CLI configure set default.output json
 
+if [ ! -r "${SSH_KEY}" ]; then
+    echo "Error: the file ${SSH_KEY} doesn't exist"
+    exit 1
+fi
+echo "[+] Uploading ssh key for the image to 's3://${S3_BUCKET_NAME}/${IMAGE_KEY}.key"
+$AWS_CLI s3 cp ${SSH_KEY} s3://${S3_BUCKET_NAME}/${IMAGE_KEY}.key --only-show-errors
```

Comment:
How are all the keys saved without overwriting each other?

Reply:
Because the `${IMAGE_KEY}` as a part of the name of the file. That variable is the name of the image and is composed with `auto-`, plus the architecture (`x86-64` or `aarch64`) and the commit hash, so it can only be one per build.
https://github.com/osbuild/automotive-ci/blob/main/tests/ci/install-vm.sh#L12

Anyway, we were discussing another option in the chat and we no longer need to upload this key, but inject a key from Testing Farm, using the secrets.